common/changes/@microsoft/rush/nickpape-dont-delete-store_2018-02-14-02-42.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Fix an issue where we always deleted the PNPM store. This is not necessary since the store is transactional. We should only delete the store if it is a --clean install.",
```

Comment:
I would have worded this as a perf optimization or feature improvement.  It's not really a bug